srcpkgs/ibus-libthai/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'ibus-libthai'
+pkgname=ibus-libthai
+version=0.1.4
+revision=1
+build_style=gnu-configure
+hostmakedepends="pkg-config"
+makedepends="ibus-devel libthai gtk+3-devel"
```

Comment:
I think `libthai-devel` would be more appropriate here. I think currently this package is pulled in as a transitive dependency via gtk3+-devel and pango-devel.